pyat/at/lattice/lattice_object.py

```diff
@@ -144,11 +161,13 @@ def __init__(self, *args,
         kwargs.setdefault('name', '')
         periodicity = kwargs.setdefault('periodicity', 1)
         kwargs.setdefault('_particle', Particle())
+        kwargs.setdefault('_fillpattern', numpy.array([1.0]))
         # Remove temporary keywords
         frequency = kwargs.pop('_frequency', None)
         cell_length = kwargs.pop('_length', None)
         cell_h = kwargs.pop('_harmnumber', math.nan)
         ring_h = kwargs.pop('harmonic_number', periodicity*cell_h)
+        self.beam_current = kwargs.pop('beam_current', 0.0)
```

Comment:
Useless: the attribute will be set in `self.update(kwargs)`, line 179

Reply:
Ok but then I have to set a default value for `beam_current`, for me it is the same...anyway it is done